eflowautodeploy/eflowAutoDeploy.ps1

```diff
@@ -917,6 +934,10 @@ function Start-EadWorkflow {
         if (!(Test-EadEflowVMSwitch -Create)) { return $false } #create switch if specified
         # We are here.. all is good so far. Validate and deploy eflow
         if (!(Invoke-EadEflowDeploy)) { return $false }
+    }
+    if (Test-EadEflowVMProvision) {
+        Write-Host "EFLOW VM is already provisioned." -ForegroundColor Yellow

```

Comment:
I would change the message - It's not really provisioned (maybe it failed), it's just the configuration was applied, or "there was an attempt to provision"

Reply:
removed this completely.